snapshots/shared/src/main/kotlin/com/emergetools/snapshots/shared/ComposePreviewSnapshotConfig.kt

```diff
@@ -0,0 +1,17 @@
+package com.emergetools.snapshots.shared
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class ComposePreviewSnapshotConfig(
+  val name: String? = null,
+  val group: String? = null,
+  val uiMode: Int? = null,
```

Comment:
nit: any type hints or docs we can link to the possible values here would be awesome